packager/packager.cc

```diff
@@ -1548,13 +1548,6 @@ vector<ast::ParsedFile> Packager::run(core::GlobalState &gs, WorkerPool &workers
 
     files = findPackages(gs, workers, std::move(files));
     setPackageNameOnFiles(gs, files);
-    if (gs.runningUnderAutogen) {
-        // Autogen only requires package metadata. Remove the package files.
-        auto it = std::remove_if(files.begin(), files.end(),
-                                 [&gs](auto &file) -> bool { return file.file.data(gs).isPackage(); });
-        files.erase(it, files.end());
-        return files;
-    }
```

Comment:
Should we `ENFORCE(!gs.runningUnderAutogen, "...")` with an explanation why?

Reply:
Sure